.github/workflows/test.yml

```diff
@@ -20,29 +20,16 @@ jobs:
         with:
           submodules: recursive
 
-      - name: Cache Qt
-        id: cache-qt
+      - name: Cache conan
         uses: actions/cache@v2.1.4
         with:
-          path: ../Qt
-          key: ${{ runner.os }}-QtCache-20201005
+          key: ${{ runner.os }}-conan-root-${{ hashFiles('**/conanfile.txt') }}
+          path: ~/.conan
 
-      # LINUX
-      - name: Install p7zip (Ubuntu)
-        if: startsWith(matrix.os, 'ubuntu')
-        run: sudo apt-get update && sudo apt-get -y install p7zip-full
-
-      - name: Install Qt
-        uses: jurplel/install-qt-action@v2
-        with:
-          aqtversion: '==1.1.1'
-          cached: ${{ steps.cache-qt.outputs.cache-hit }}
-          extra: --external 7z
-
-      # LINUX
       - name: Install dependencies (Ubuntu)
         if: startsWith(matrix.os, 'ubuntu')
         run: |
+            curl "https://raw.githubusercontent.com/AnotherFoxGuy/ci-scripts/main/install-conan.sh" | sudo bash
```

Comment:
we should not be using conan when building the tests, or we should be using both conan and not conan through the matrix function

and don't run a random curl shell script to install conan when it's so few lines on ubuntu
```
      - name: Install Conan (Ubuntu)
        run: |
            sudo apt-get update && sudo apt-get -y install python3 python3-pip python3-setuptools
            pip3 install --user conan
            echo "$HOME/.local/bin" >> $GITHUB_PATH
```